test/test_util/example_config.yaml

```diff
@@ -0,0 +1,29 @@
+---
+version: 1
+disable_existing_loggers: False
+formatters:
+  simple:
+    format: '[%(levelname)s] [%(asctime)s:%(name)s] %(message)s'
+
+handlers:
+  console:
+    class: logging.StreamHandler
+    level: WARNING
+    formatter: simple
+    stream: ext://sys.stdout
+
+  file_handler:
+    class: logging.FileHandler
+    level: DEBUG
+    formatter: simple
+    filename: automl.log
+
+  distributed_logfile:
+    class: logging.FileHandler
+    level: DEBUG
+    formatter: simple
+    filename: distributed.log
+
+root:
+  level: CRITICAL
+  handlers: [console, file_handler]
```

Comment:
This previously used loggers, too. Could you briefly explain why there are no more loggers in here?

Reply:
The other loggers are for auto-sklearn exclusively? I put this file here as an example. We can even delete this file and have each repository create this as needed?

Reply:
There were some loggers for SMAC, maybe we can use them here?

Reply:
Done!